src/Model/Item/items.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Billogram\Model\Item;
+
+use Billogram\Exception\InvalidArgumentException;
+use Billogram\Model\CreatableFromArray;
+
+/**
+ * @author Ibrahim Hizeoui <ibrahimhizeoui@gmail.com>
+ */
+class items implements CreatableFromArray
```

Comment:
Classes must have their first char as capital. Same with files. 

Also, classes should be singular